src/composables/mvt-styles/mvt-styles.composable.ts

```diff
@@ -293,11 +294,15 @@ export default function useMvtStyles() {
     bgLayer: Layer | undefined | null
   ): StyleCapabilities {
     const bgLayerDef = getBgLayerDef(bgLayer)
+    const styleEditors: StyleSection[] = [
+      ...(bgLayerDef?.simple_style_class ? [StyleSection.simpleStyle] : []),
+      ...(bgLayerDef?.medium_style_class ? [StyleSection.mediumStyle] : []),
+      ...(bgLayerDef?.expert_style_class ? [StyleSection.advancedStyle] : []),
+    ]
+
```

Comment:
I had a hard time understanding this refactoring.

I'm more used to binary operations like
```
export enum StyleSection {
  simpleStyle = ( 1 <<0 ),
  mediumStyle = ( 1 << 1 ),
  advancedStyle = ( 1 << 2 ),
}
styleEditors = (
  bgLayerDef?.simple_style_class ? StyleSection.simpleStyle : 0
| bgLayerDef?.medium_style_class ? StyleSection.mediumStyle : 0
| bgLayerDef?.expert_style_class ? StyleSection.advancedStyle : 0
)
```
hasMedium = styleEditors & StyleSection.mediumStyle

this will be more efficient in execution than using a list for available editors (which does not prevent multiple identical identifiers, the list could contain 2x simple editor) and variable expansion

Reply:
```
export enum StyleSection {
  simpleStyle = ( 1 <<0 ),
  mediumStyle = ( 1 << 1 ),
  advancedStyle = ( 1 << 2 ),
}
```
really????

Reply:
I did not check how to define binary enum in typescript, this might be a syntax error, it is the way it is done for mutually exclusive constants in libraries.

Reply:
How do I select the first one panel to open by default then?

Reply:
I will merge this PR.
Feel free to rework this in another ticket if you think it is worth it.